.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F31F User Story"
+about: Create a user story to statify a feature request.
```

Comment:
Is "statify" a word? Can't find it in my dictionary :)

Reply:
IT'S CANADIAN ENGLISH OKAY!

Reply:
Fixed. 😛